apps/election-manager/src/components/ExportFinalResultsModal.test.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react'
+
+import { fireEvent, waitFor } from '@testing-library/react'
+import fetchMock from 'fetch-mock'
+import MockDate from 'mockdate'
+
+import { UsbDriveStatus } from '../lib/usbstick'
+import ExportFinalResultsModal from './ExportFinalResultsModal'
+import fakeKiosk, { fakeUsbDrive } from '../../test/helpers/fakeKiosk'
+import renderInAppContext from '../../test/renderInAppContext'
+
+MockDate.set(new Date(2020, 2, 14, 1, 59, 26))
```

Comment:
Put this in a `beforeEach` and a `MockDate.reset()` in an `afterEach`? Or just put it in the test that needs it?